packages/amplify-e2e-tests/__tests__/storage.test.ts

```diff
@@ -0,0 +1,85 @@
+require('../src/aws-matchers/'); // custom matcher for assertion
+import { initProjectWithProfile, deleteProject, amplifyPushAuth } from '../src/init';
+import { addAuthWithDefault } from '../src/categories/auth';
+import { addSimpleDDB, addDDBWithTrigger, updateDDBWithTrigger, addS3WithTrigger } from '../src/categories/storage';
+import { createNewProjectDir, deleteProjectDir, getProjectMeta, getDDBTable, checkIfBucketExists } from '../src/utils';
+
+
+describe('amplify add/update storage(S3)', () => {
+  let projRoot: string;
+  beforeEach(() => {
+    projRoot = createNewProjectDir();
+    jest.setTimeout(1000 * 60 * 60); // 1 hour
+  });
+
+  afterEach(async () => {
+    await deleteProject(projRoot);
+    deleteProjectDir(projRoot);
+  });
+
+  it('init a project and add S3 bucket with trigger', async () => {
+    await initProjectWithProfile(projRoot, {});
+    await addAuthWithDefault(projRoot, {});
+    await addS3WithTrigger(projRoot, {});
```

Comment:
Maybe have a method to call the Lambda API and check that the function exists?

Reply:
I’ll add it once you merge your PR.